core/src/main/java/org/testcontainers/containers/ContainerState.java

```diff
@@ -43,6 +43,22 @@ default boolean isRunning() {
         }
     }
 
+    /**
+     * @return is the container created?
+     */
+    default boolean isCreated() {
+        if (getContainerId() == null) {
+            return false;
+        }
+
+        try {
+            Boolean created = getCurrentContainerInfo().getState().getStatus().equalsIgnoreCase("Created");
+            return Boolean.TRUE.equals(created);
```

Comment:
In the `isRunning` method the result of `getRunning()` is a nullable `Boolean`, which is why we do the `TRUE.equals` call in that method.

Here, `equalsIgnoreCase` just returns a primitive boolean, and further, `getStatus()` may return null.

As such, I think it would actually be safer to just replace these two lines with:
`return "created".equalsIgnoreCase(getCurrentContainerInfo().getState().getStatus());`

Please could we do that instead?

Reply:
Also, this method will return `false` if the container is not just created but also started